syntaxes/vba.yaml-tmlanguage

```diff
@@ -92,9 +93,9 @@ repository:
         match: (?i)\b(Class|Const|Dim|(G|L|S)et|ReDim( Preserve)?|Erase|Static)\b
 
   strings:
-    name: string.quoted.double
-    begin: '"'
-    end: '"'
+    patterns:
+    - name: string.quoted.double
+      match: (?i)"[^\n\r\"]*"
```

Comment:
As far as I remember, matching across multiple lines is not supported. So the rule should already do the same as your change, while it's a lot simpler.

I will however verify this again.

And in general `(?i)` is not needed here. It disables case-sensitivity for letters.

Reply:
Thank you for remarks.